src/main/java/fi/vm/yti/terminology/api/importapi/simpleexcel/SimpleExcelParser.java

```diff
@@ -0,0 +1,232 @@
+
+                            var cellValue = row.getCell(headers.get(entry.getKey())).getStringCellValue();
+
+                            //Multiline properties can have multiple properties per cell
+                            if (Arrays.asList(CONCEPT_MULTILINE_PROPERTIES).contains((propertyName))) {
+                                cellValue.lines()
+                                        //lines() can output empty lines but not null lines
+                                        .filter(String::isEmpty)
+                                        //Add all lines as a new attribute
+                                        .forEach(value -> {
+                                            if (isPropertyValid(propertyName, value)) {
+                                                attributes.add(new Attribute(lang, value));
+                                            } else {
+                                                throw new ExcelParseException("value-not-valid", row, entry.getValue());
+                                            }
+                                        });
+                            } else {
+                                //Add the whole string as attribute
+                                if (isPropertyValid(propertyName, cellValue)) {
+                                    attributes.add(new Attribute(lang, cellValue));
+                                } else {
+                                    throw new ExcelParseException("value-not-valid", row, entry.getValue());
+                                }
+                            }
+                            properties.put(propertyName, attributes);
+                        }
+                );
+        return properties;
+    }
+
+    /**
+     * Create all terms
+     *
+     * @param row           Row to check
+     * @param headers       Headers
+     * @param terminologyId Terminology id
+     * @param status        Concept status
+     * @return Map of reference names and their nodes
+     */
+    private Map<String, List<GenericNode>> createTerms(Row row, Map<String, Integer> headers, UUID terminologyId, String status) {
+        Map<String, List<GenericNode>> terms = new HashMap<>();
+        headers.entrySet().stream()
+                .filter(entry -> cellHasText(row.getCell(headers.get(entry.getKey()))) &&
+                        Arrays.asList(TERM_TYPES).contains(entry.getKey().split(HEADER_SEPARATOR)[0]))
+                .forEach(entry -> {
+                    String[] headerValues = entry.getKey().split(HEADER_SEPARATOR);
+                    if (headerValues.length != 2) {
+                        throw new ExcelParseException("term-missing-language-suffix", row, entry.getValue());
+                    }
+                    List<GenericNode> termList = terms.getOrDefault(headerValues[0], new ArrayList<>());
+
+                    row.getCell(headers.get(entry.getKey()))
+                            .getStringCellValue().lines()
```

Comment:
concept can have only one preferable term (=no multiline possibility in prefLabel_language column)